src/main/java/io/confluent/connect/elasticsearch/Validator.java

```diff
@@ -269,11 +269,10 @@ private void validateConnection(RestHighLevelClient client) {
     String exceptionMessage = "";
     try {
       successful = client.ping(RequestOptions.DEFAULT);
-    } catch (IOException e) {
+    } catch (IOException | RuntimeException e) {
```

Comment:
```suggestion
    } catch (Exception e) {
```

we might as well make it the most generic